README.md

```diff
@@ -35,38 +32,162 @@ npx openapi-typescript https://petstore.swagger.io/v2/swagger.json --output pets
 # 🚀 https://petstore.swagger.io/v2/swagger.json -> petstore.ts [650ms]
 ```
 
-**Typed fetch client**
+### Using SvelteFetcher
+
+Configure SvelteFetcher instance and generate functions for making API calls:
 
 ```ts
-import { Fetcher } from 'openapi-typescript-fetch'
+// File: api.ts
 
+import { SvelteFetcher } from '@cocreators-ee/apity'
```

Comment:
Your import says `SvelteFetcher` but usage below is `Fetcher`, this will be fixed when you rename it to `Apity` :)